pkg/maps/lxcmap/lxcmap.go

```diff
@@ -192,10 +193,20 @@ func WriteEndpoint(f EndpointFrontend) error {
 // AddHostEntry adds a special endpoint which represents the local host
 func AddHostEntry(ip net.IP) error {
 	key := NewEndpointKey(ip)
-	ep := EndpointInfo{Flags: EndpointFlagHost}
+	ep := &EndpointInfo{Flags: EndpointFlagHost}
 	return LXCMap.Update(key, ep)
 }
 
+func SyncHostEntry(ip net.IP) error {
+	key := NewEndpointKey(ip)
+	value, err := LXCMap.Lookup(key)
+	if err != nil || value.(*EndpointInfo).Flags&EndpointFlagHost == 0 {
+		log.WithError(err).WithField(logfields.IPAddr, ip).Debugf("Adding local ip to endpoint map")
```

Comment:
There's nothing that is specific about this being a `local ip` here, perhaps we can shift this to the caller.

Reply:
I would prefer to return an error here and log in the caller

Reply:
Done.